siftseek/endpoints/seeker/profile/seeker_profile_delete.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from flask import Response, jsonify
+
+from siftseek.models.db import db
+from siftseek.models.seeker import Seeker
+from siftseek.endpoints.seeker import seeker
+from siftseek.endpoints.helpers.query_helpers import get_model_by_pk_or_404
+
+
+@seeker.delete("/profile/<int:seeker_id>")
+def mark_for_deletion(seeker_id: int) -> tuple[Response, int]:
+    """
+    Marks a profile for deletion. Actual deletion is handled by a chron job in
```

Comment:
Why is the actual deletion handled by a cron job? 
Is it necessary to do a hard delete via cron job?

Reply:
The idea behind this was that the delete, from the user perspective, will act more like a deactivate, and they can change their mind within some period. I just defaulted that period to 30 days, but on a final pass would have to check for GDPR compliance.

Since this project is supposed to be a portfolio piece showing off what I know more so than trying to create something that I actually expect people to use, I was just taking this opportunity to show that I know about cron jobs and make the delete method a little more advanced than just a simple hard delete.